media/js/base/site.js

```diff
@@ -155,16 +109,11 @@
         var _version = version ? parseFloat(version) : 0;
 
         if (platform === 'windows') {
-            // Add class to allow Windows XP/Vista users to download Firefox 52 ESR, though these
-            // legacy platforms are now officially unsupported
+            // Add class for Windows XP/Vista users to display
+            // unsupported messaging on /download/thanks/ page.
             if (_version >= 5.1 && _version <= 6) {
                 h.className += ' xpvista';
             }
-
-            // Add class to support downloading Firefox for Windows 64-bit on Windows 7 and later
```

Comment:
64bit downloads for Windows have never been enabled since the stub installer can handle these checks more accurately than we can do using UA, so removing the logic.